controllers/nonadmin_controller.go

```diff
@@ -86,6 +85,21 @@ func (r *DPAReconciler) ReconcileNonAdminController(log logr.Logger) (bool, erro
 		return true, nil
 	}
 
+	selector, err := fields.ParseSelector(fmt.Sprintf("metadata.namespace!=%s", r.NamespacedName.Namespace))
+	if err != nil {
+		return false, err
+	}
+	dpaList := &oadpv1alpha1.DataProtectionApplicationList{}
+	err = r.List(r.Context, dpaList, &client.ListOptions{FieldSelector: selector})
+	if err != nil {
+		return false, err
+	}
+	for _, dpa := range dpaList.Items {
+		if (&DPAReconciler{dpa: &dpa}).checkNonAdminEnabled() {
+			return false, fmt.Errorf("only one NAC can be installed in the whole cluster")
```

Comment:
`Only a single instance of Non-Admin Controller can be installed across the entire cluster. Non-Admin controller is already deployed in <NAC-Enabled-DPA.NS>`